### DISCUSSION TITLE
Help understading generated shop types

### DISCUSSION THREAD

**Discussion by proudchild** [vendurehq/vendure#1192] [Q&A]
Hello All!

Followed the example on real-world-vendure and created a plugin, but when i run yarn start it throw many errors on types generated-shop-types.

On the reviews example the files are there, on my version had to change all types to ANY and i still dont understand how to generate those files:

the files i mean are the following:
https://github.com/vendure-ecommerce/real-world-vendure/blob/master/src/plugins/reviews/generated-admin-types.ts
https://github.com/vendure-ecommerce/real-world-vendure/blob/master/src/plugins/reviews/generated-shop-types.ts

do i have to code those? if not, how do i generate them?

**Comment by michaelbromley** [comment]
This is mentioned in the repo's readme: https://github.com/vendure-ecommerce/real-world-vendure#graphql-code-generator
